src/gui/editors/PianoRoll.cpp

```diff
@@ -677,30 +677,36 @@ void PianoRoll::glueNotes()
 		// Make undo possible
 		m_pattern->addJournalCheckPoint();
 
+		// Sort notes on key (default it's sorted on pos, so after this
+		// it is sorted on key and then pos)
+		std::sort(selectedNotes.begin(), selectedNotes.end(),
+											Note::compareKey);
```

Comment:
I'm on the fence about having `compareKey` as a new function in `Note`.

I almost think it would be better to put a lambda function here.

Reply:
I'm ok using lambda function there, changed it.